src/App.js

```diff
@@ -7,14 +7,45 @@ import {
 import Books from './pages/Books';
 import Categories from './pages/Categories';
 
+import SvgUser from './imageComponents/UserIcon.tsx';
+
 export default function App() {
+  const [isBooksActive, setIsBooksActive] = useState(false);
+  const [isCategoriesActive, setIsCategoriesActive] = useState(false);
+
+  const handleBooksClick = () => {
+    setIsBooksActive(true);
+    setIsCategoriesActive(false);
+  };
+
+  const handleCategoriesClick = () => {
+    setIsBooksActive(false);
+    setIsCategoriesActive(true);
+  };
   return (
     <BrowserRouter>
       <header>
-        <nav>
-          <li>Bookstore CMS</li>
-          <NavLink to="" activeclassname="active-link" className>Books</NavLink>
-          <NavLink to="categories" activeclassname="active-link">Categories</NavLink>
+        <nav className="panel-bg">
+          <li className="Bookstore-CMS">Bookstore CMS</li>
+          <NavLink
+            to=""
+            activeClassName="active-link"
+            className={`BOOKS ${isBooksActive ? 'active-link' : ''}`}
+            onClick={handleBooksClick}
+          >
+            Books
+          </NavLink>
+          <NavLink
+            to="categories"
+            activeClassName="active-link"
+            className={`CATEGORIES ${isCategoriesActive ? 'active-link' : ''}`}
+            onClick={handleCategoriesClick}
+          >
+            Categories
+          </NavLink>
+          <div className="oval">
+            <SvgUser className="Mask" />
+          </div>
         </nav>
```

Comment:
- [ ] Your navbar component is a great addition to your app; however, it currently doesn't align with the provided design. To enhance your app's aesthetics and user experience, please ensure that you use the specified layout to match the design. Please ensure that the navbar is full with the book's card. By incorporating these design elements, you'll bring your navbar closer to the intended design, improving the overall look and feel of your app. Keep up the good work! 👍🎨

| Your | Design |
| --- | --- |
|![image](https://github.com/eamador1/bookstore-redux/assets/55840999/b2d3cf0e-a975-4f02-b700-1092fcbf8b75)|  ![image](https://github.com/eamador1/bookstore-redux/assets/55840999/64a3fa05-b200-4517-841d-2a828132c1a6)|